src/dotnet/APIView/APIView/Model/TokenTreeModel.cs

```diff
@@ -0,0 +1,182 @@
+using System.Collections.Generic;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+
+namespace csharp_api_parser.TreeToken
+{
+    public enum StructuredTokenKind
+    {
+        Content = 0,
+        LineBreak = 1,
+        NoneBreakingSpace = 2,
+        TabSpace = 3,
+        ParameterSeparator = 4,
+        Url = 5
+    }
+
+
+    public class StructuredToken
+    {
+        public string Value { get; set; } = string.Empty;
+        public string Id { get; set; }
+        public StructuredTokenKind Kind { get; set; }
```

Comment:
```suggestion
        public StructuredTokenKind Kind { get; set; } = StructuredTokenKind.Content
```

Reply:
Why is this necessary?

Reply:
What is the default Kind? or Is it always set for all tokens?